views/base.html

```diff
@@ -4,6 +4,8 @@
     <title>{{title or "PyJudge"}}</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="/static/css/bootstrap.min.css">
+    <link rel="stylesheet" href="/static/css/notfound.css">
```

Comment:
base file needs this?

Reply:
yes this css file is require for the notfound template thats why i added it here

Reply:
In the code i have written this message:
The page you are looking for might have been removed had its name changed or is temporarily unavailable

Reply:
if you want then i can remove the css for now and design basic html only??

Reply:
ah no. Keep the css. I think including the 404 css in the base template is wrong. this needs to be added to the 404 template. Perhaps if you did not use `include` in the 404 template and instead used `rebase`?

Reply:
should i use rebase?? instead of include??

Reply:
i think i should remove the include and make it a normal html with css included?? is it fine??

Reply:
no. please use the template system.

Reply:
no. please use the template system.